The asadmin start-database command needs to include derbyshared.jar on the classpath for JDK11 support

`asadmin start-database` needs to be updated to include `derbyshared.jar`.  This is important to resolve a class not found failure when starting the Derby server in the JPA TCK.

## Environment Details
* GlassFish Version (and build number): master branch as of March 12, 2021
* JDK version: 11
* OS: Linux
* Database: Derby

----------

## Problem Description
In order to completely solve https://github.com/eclipse-ee4j/jakartaee-tck/issues/627, we need the `./asadmin start-database --dbhome $TCK_HOME/$GF_TOPLEVEL_DIR/glassfish/databases --dbport 1527` command to include the new `derbyshared.jar` on the classpath to avoid the `Class Not Found: JDBC driver org.apache.derby.jdbc.EmbeddedDriver could not be loaded` failure.
[//]: # "State if the problem is easily reproducible or happens intermittently"
It is easy to reproduce in the [Jakarta EE TCK Jenkins test environment ](https://ci.eclipse.org/jakartaee-tck/)
[//]: # "Include stack traces or command outputs"
```
init.derbyEmbedded:
     [echo] ts.home = /root/jakartaeetck/bin/xml/../..
     [echo] In a bundle
     [echo] The deliverable currently in use is /root/jakartaeetck/bin/xml/../../bin.

check.configure.cmp:

configure.basic:
     [echo] DB URL: jdbc:derby:/tmp/DerbyDB;create=true

BUILD FAILED
/root/jakartaeetck/bin/build.xml:441: The following error occurred while executing this line:
/root/jakartaeetck/bin/xml/impl/glassfish/javaee_vi.xml:107: The following error occurred while executing this line:
/root/jakartaeetck/bin/xml/initdb.xml:137: The following error occurred while executing this line:
/root/jakartaeetck/bin/xml/initdb.xml:505: Class Not Found: JDBC driver org.apache.derby.jdbc.EmbeddedDriver could not be loaded
```

## Steps to reproduce
[//]: # "Step by step instructions to reproduce the problem"
[//]: # "Provide sample code/application if relevant"
I tried reproducing locally with JDK11 + running `asadmin start-database --dbport 1527` but that didn't reproduce.  So, I'm not sure of what the JPA TCK is doing differently.

Next, I'll try the same command that the tck uses `asadmin start-database --dbhome /tmp/derbydatabases --dbport 1527 --jvmoptions "-Djava.security.manager -Djava.security.policy=$TS_HOME/bin/security.policy"`

## Impact of Issue
[//]: # "How has this issue affected you? What are you trying to accomplish?"
This prevents us from passing the Standalone JPA TCK and possibly other TCKs.
[//]: # "Please choose one each Type, Component, and Priority label"
ee9.1-tck

--- a/appserver/admin/cli-optional/src/main/java/com/sun/enterprise/admin/cli/optional/DatabaseCommand.java
+++ b/appserver/admin/cli-optional/src/main/java/com/sun/enterprise/admin/cli/optional/DatabaseCommand.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1997, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1997, 2021 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
  * terms of the Eclipse Public License v. 2.0, which is available at
@@ -76,6 +76,7 @@ public abstract class DatabaseCommand extends CLICommand {
 	sClasspath.add(new File(sInstallRoot, "lib/asadmin/cli-optional.jar"));
         sDatabaseClasspath
                 .add(dbLocation,"lib","derby.jar")
+                .add(dbLocation,"lib","derbyshared.jar")
                 .add(dbLocation,"lib","derbytools.jar")
                 .add(dbLocation,"lib","derbynet.jar")
                 .add(dbLocation,"lib","derbyclient.jar");

